Cargo.toml

```diff
@@ -38,6 +38,7 @@ required-features = ["json-logger"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
 opentelemetry = {version = "0.31", optional = true}
+opentelemetry-semantic-conventions = {version = "0.31", optional = true}
```

Comment:
We could probably remove this dependency since we have removed the use of semantic conventions.